Remove obsolete technical roadmap page (#649)
medic/cht-docs: content/en/core/overview/roadmap.md

---
title: "Roadmap"
linkTitle: "Roadmap"
weight: 1
description: >
  Improvements that are being worked on, and those that are being considered next
aliases:
  -    /core/overview/technical-roadmap
relatedContent:
  core/releases
---

{{% pageinfo %}}

> The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht#the-cht-community" %}}).

Reviewer: I'm on the fence between "actively developed" and "actively maintained and developed", but leaned towards the former by [borrowing language](https://www.repostatus.org/#active). 
```suggestion
The CHT is actively developed by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht#the-cht-community" %}}).
```

> The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht/#the-cht-community" %}}).

Author: Remove the slash to fix the link and the build.

```suggestion
The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht#the-cht-community" %}}).
```

> The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht/#the-cht-community" %}}).

Author: Do you think this wording is more consistent? It also sounds better to me than maintained which may imply the CHT is not being developed further.

```suggestion
The CHT is stewarded by [Medic](https://medic.org), a non-profit organization alongside many [partners in the CHT Community]( {{% ref "why-the-cht/#the-cht-community" %}}).
```

Reviewer: Right, I see what you mean. I find "stewarded" awkward, and not entirely clear, so will make another suggestion. I'll have to make it separately because this thread is marked _outdated_ by accepting the previous suggestion.
